libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/store/EventStoreManager.java

```diff
@@ -236,64 +224,11 @@ public int getMaxEvents() {
      * @return Number of stored events.
      */
     public int getNumStoredEvents() {
-        int numFiles = 0;
-        final File[] listOfFiles = rootDir.listFiles();
-        if (listOfFiles != null) {
-            numFiles = listOfFiles.length;
-        }
-        return numFiles;
+        return book.getAllKeys().size();
     }
 
     private boolean shouldStoreEvent() {
-        final List<File> files = getAllFiles();
-        int fileCount = 0;
-        if (files != null) {
-            fileCount = files.size();
-        }
-        return (isLoggingEnabled && (fileCount < maxEvents));
-    }
-
-    private InstrumentationEvent fetchEvent(File file) {
```

Comment:
We don't need this anymore - fetching an event is straightforward.

Reply:
🎉